src/__tests__/Components.react.test.js

```diff
@@ -0,0 +1,156 @@
+import { render, within } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import { MemoryRouter } from 'react-router-dom';
+
+import Card from '../components/Card';
+import DetailsCard from '../components/DetailsCard';
+import Header from '../components/Header';
+import Hero from '../components/Hero';
+import SearchBar from '../components/SearchBar';
+
+describe('Card', () => {
+  test('should render component', () => {
+    const { asFragment } = render(
+      <Card
+        id="AAPL"
+        ticker="AAPL"
+        changes={1.2}
+        price="250"
+        changesPercentage="0.23"
+        companyName="Apple Inc."
+      />,
+      { wrapper: MemoryRouter },
+    );
+    expect(asFragment()).toMatchSnapshot();
+  });
```

Comment:
- [ ] The test on rendering the Card component is failing. Kindly fix that

![image](https://user-images.githubusercontent.com/50941074/189502673-bb024b8e-ffe4-45dd-834b-3325d30aa27c.png)